content/pt-br/docs/tasks/access-application-cluster/access-cluster.md

```diff
@@ -0,0 +1,269 @@
+
+curl $APISERVER/api --header "Authorization: Bearer $TOKEN" --insecure
+```
+
+O resultado é semelhante a este:
+
+```json
+{
+  "kind": "APIVersions",
+  "versions": [
+    "v1"
+  ],
+  "serverAddressByClientCIDRs": [
+    {
+      "clientCIDR": "0.0.0.0/0",
+      "serverAddress": "10.0.1.149:443"
+    }
+  ]
+}
+```
+
+Usando `jsonpath`:
+
+```shell
+APISERVER=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
+TOKEN=$(kubectl get secret default-token -o jsonpath='{.data.token}' | base64 --decode)
+
+curl $APISERVER/api --header "Authorization: Bearer $TOKEN" --insecure
+```
+
+O resultado é semelhante a este:
+
+```json
+{
+  "kind": "APIVersions",
+  "versions": [
+    "v1"
+  ],
+  "serverAddressByClientCIDRs": [
+    {
+      "clientCIDR": "0.0.0.0/0",
+      "serverAddress": "10.0.1.149:443"
+    }
+  ]
+}
+```
+
+Os exemplos acima usam a opção `--insecure`. Isso o deixa sujeito a ataques MITM. 
+Quando o kubectl acessa o cluster, ele usa um certificado raiz guardado
+e certificados de cliente para acessar o servidor. (Esses certificados são instalados no diretório
+`~/.kube`). Como os certificados do cluster normalmente são autoassinados, pode ser necessária uma
+configuração especial para que seu cliente http use o certificado raiz.
+
+Em alguns clusters, o apiserver não requer autenticação; ele pode servir
+no localhost ou estar protegido por um firewall. Não há um padrão
+para isso. [Controle de acesso à API](/docs/concepts/security/controlling-access) 
+descreve como um administrador de cluster pode configurar isso.
+
+## Acesso programático à API
+
+O Kubernetes suporta oficialmente as bibliotecas de clientes [Go](#go-client) e [Python](#python-client).
+
+### Biblioteca Go client
+
+* Para obter a biblioteca, execute o seguinte comando: `go get k8s.io/client-go@kubernetes-<kubernetes-version-number>`,
+  consulte [INSTALL.md](https://github.com/kubernetes/client-go/blob/master/INSTALL.md#for-the-casual-user)
+  para obter instruções detalhadas de instalação. Consulte
+  [https://github.com/kubernetes/client-go](https://github.com/kubernetes/client-go#compatibility-matrix)
+  para ver quais versões são compatíveis.
+* Escreva um aplicativo utilizando a biblioteca Go Client. Observe que ela define seus próprios objetos de API,
+  portanto, se necessário, importe as definições de API da biblioteca Go Client em vez de importá-las do repositório principal.
+  Por exemplo, `import "k8s.io/client-go/kubernetes"` está correto.
+
+A biblioteca Go Client pode usar o mesmo arquivo [kubeconfig] (/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
+como a CLI do kubectl faz, para localizar e autenticar ao apiserver. Veja esse
+[exemplo](https://git.k8s.io/client-go/examples/out-of-cluster-client-configuration/main.go).
+
+Se o aplicativo for disponibilizado como um pod no cluster, consulte a [próxima seção] (#acessar-o-api-deum-pod).
+
+### Bliblioteca Python client
```

Comment:
Hi, may you take a look here there is a typo "Bliblioteca", change to "Biblioteca", thanks.